mani_skill/envs/sapien_env.py

```diff
@@ -620,9 +620,14 @@ def _setup_sensors(self, options: dict):
         # Cameras for rendering only
         self._human_render_cameras = dict()
         for uid, camera_cfg in self._human_render_camera_configs.items():
+            if uid in self._agent_sensor_configs:
+                articulation = self.agent.robot
+            else:
+                articulation = None
             self._human_render_cameras[uid] = Camera(
                 camera_cfg,
                 self.scene,
+                articulation=articulation,
```

Comment:
actually we should be deprecating the use of a link string id to specify where to mount as it requires the implicit assumption that we fill in the correct articulation to look for the corresponding link string id.

Better code is to directly reference the link object in the camera config (see later comments). Hence these new lines should be removed.

Reply:
fixed